### PR TITLE
Replace all instances of double-caret uppercase-expansion with pipes to tr

### PR DESCRIPTION
**Pull request by hardyoyo**
- Better support for versions of Bash that do not support double-caret
  uppercase expansion
- Specifically gets Powerbash working on a Homebrew-packaged installation of bash version
  5.2.21(1)

### CODE REVIEW

File: powerbash.sh
Comment (napalm255) [inline powerbash.sh:1]: Appears the indenting is off.

**Comment by hardyoyo** [comment]
I'm inclined to close this, I have tracked down a weird bug that presents only with this change in place: if I hit ctrl-C very quickly, after three presses, my shell exits, if I'm also running powerbash. I think this is because one of the ctrl-C interrupts sucessfully cancels the tr command in the Powerbash shell... and thus cancels the entire shell. Maybe not a deal killer for everyone, but definitely is for me.

I'd love for Powerbash to work with my version of Bash (absolute latest 5.2.21) but for some reason the code identified and replaced in this PR (double-caret uppercase-expansion) fails to work.

TLDR: This PR is flawed, I am closing it now. Powerbash does not work with Bash 5.2.21.
